.kokoro/runintegrationtests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+# Build the libraries and run unit tests.
+./build.sh
+
+# Allow each integration test 3 chances to pass.
+./runintegrationtests.sh || true
+./runintegrationtests.sh --retry || true
```

Comment:
Does it matter that this is running `runintegrationtests.sh` three times, even if the first run passes?

Reply:
No, it will run tests for all entries in the failure file.  Which will be empty if everything passes.  We could probably make this a little more efficient (not having to run some checks in a shell script) but at the cost of complexity.